ts/components/PinCreationForm.tsx

```diff
@@ -159,6 +159,7 @@ export const PinCreationForm = ({ onSubmit }: Props) => {
             isValid={isPinValid ? undefined : false}
             overrideBorderColor={isPinValid ? undefined : IOColors.red}
             testID="PinField"
+            inputAccessoryViewID="" // hide keyboard toolbar
```

Comment:
Why should we hide the keyboard accessory toolbar?

Reply:
<img src="https://github.com/pagopa/io-app/assets/114305019/3cc3f34b-284b-4dae-a8b2-5e5df09a6953" width = 350>

I guess we should remove the keyboard toolbar because it's actually empty and takes up some space